tools/linter_lib/custom_check.py

```diff
@@ -415,33 +415,6 @@
     shebang_rules=shebang_rules,
 )
 
-css_rules = RuleList(
-    langs=['css', 'scss'],
-    rules=[
-        {'pattern': r'^(?:(?!/\*).)*https?://',
-         'description': "Zulip CSS should have no dependencies on external resources",
-         'good_lines': ['background: url(/static/images/landing-page/pycon.jpg);'],
-         'bad_lines': ['background: url(https://example.com/image.png);']},
-        {'pattern': ' thin[ ;]',
-         'description': "thin CSS attribute is under-specified, please use 1px.",
-         'good_lines': ["border-width: 1px;"],
-         'bad_lines': ["border-width: thin;", "border-width: thin solid black;"]},
-        {'pattern': ' medium[ ;]',
-         'description': "medium CSS attribute is under-specified, please use pixels.",
-         'good_lines': ["border-width: 3px;"],
-         'bad_lines': ["border-width: medium;", "border: medium solid black;"]},
-        {'pattern': ' thick[ ;]',
-         'description': "thick CSS attribute is under-specified, please use pixels.",
-         'good_lines': ["border-width: 5px;"],
-         'bad_lines': ["border-width: thick;", "border: thick solid black;"]},
-        {'pattern': r'rgba?\(',
-         'description': 'Use of rgb(a) format is banned, Please use hsl(a) instead',
-         'good_lines': ['hsl(0, 0%, 0%)', 'hsla(0, 0%, 100%, 0.1)'],
-         'bad_lines': ['rgb(0, 0, 0)', 'rgba(255, 255, 255, 0.1)']},
-        *whitespace_rules,
-    ],
-)
```

Comment:
We might want to preserve `css_rules` as an empty list in case we later add Zulip-specific rules (E.g. names of this form must be in this file, or whatever)?

Reply:
Meh? Added it back, but it’s not like it would have been a barrier to do that on demand.